hash/nuon.xml

```diff
@@ -7,78 +7,96 @@ license:CC0-1.0
 <softwarelist name="nuon" description="NUON DVD-ROMs">
 
 	<software name="ballstic">
-		<description>Ballistic</description>
+		<description>Ballistic (USA, Europe)</description>
 		<year>2000</year>
 		<publisher>VM Labs, Inc.</publisher>
 		<part interface="dvdrom" name="dvdrom">
 			<diskarea name="dvdrom">
-				<disk name="ballstic" sha1="92b386f176eecc7d3b1e13675ccaf0b60a543175" />
+				<disk name="ballistic (usa, europe)" sha1="5371a3abfb92bc1841d3416938bc07b2619024f0" />
 			</diskarea>
 		</part>
 	</software>
 
 	<software name="freefall">
-		<description>FreeFall 3050 A.D.</description>
-		<year>200?</year>
+		<description>FreeFall 3050 A.D. (USA, Europe)</description>
+		<year>2000</year>
 		<publisher>VM Labs, Inc.</publisher>
 		<part interface="dvdrom" name="dvdrom">
 			<diskarea name="dvdrom">
-				<disk name="freefall" sha1="56540609dcdf3d5234a1d045b28cba58ba6adae8" />
+				<disk name="freefall 3050 a.d. (usa, europe)" sha1="5bbff6c8576a97a6cb5deabce59de4280157df5a" />
 			</diskarea>
 		</part>
 	</software>
 
 	<software name="is3">
-		<description>Iron Soldier 3</description>
+		<description>Iron Soldier 3 (USA)</description>
+		<year>2001</year>
+		<publisher>VM Labs, Inc.</publisher>
+		<part interface="dvdrom" name="dvdrom">
+			<diskarea name="dvdrom">
+				<disk name="iron soldier 3 (usa)" sha1="5e8951552c80fb0148de33fd2c77c29a9a769603" />
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="jm3">
```

Comment:
If the SW item is "not working" then it should have `supported="no"` flag (this applies to everything in this file really).

Reply:
Gotcha. I'll add the tag to all the items later today.

Reply:
A lot of the software lists for skeleton systems, or systems marked not working where it’s implied nothing works, don’t bother setting `supported` attributes.

Reply:
In my crusade against `supported=<undefined>` (sometimes with rather [embarrassing later demotions](https://github.com/mamedev/mame/commit/a98cf345cd7d6d0370f69b25c5bcb85de80258ea)) hearing that a 6th gen DVD player with likely performance and structural pitfalls don't need `supported=no` isn't an option, it will be done either in this PR or later by me. It just won't happen that anybody will emulate this in one big commit, and it's baby regex that requires 30 seconds.